blog/_posts/2024-06-25-history-ele-raid-buffs.md

```diff
@@ -145,18 +145,21 @@ Name | 🔥Benefits
 Mists of Pandaria is the expansion where we started to see a decline in Shaman raid utility, with Elemental and Restoration hit particularly hard.
 Mists brought with it a massive pruning of Shaman totems that had historically provided a number of buffs to raids.
 Totems like [Strength of Earth Totem](https://wowpedia.fandom.com/wiki/Strength_of_Earth_Totem), [Windfury Totem](https://wowpedia.fandom.com/wiki/Windfury_Totem), and [Wrath of Air Totem](https://wowpedia.fandom.com/wiki/Wrath_of_Air_Totem) were removed entirely, while other like [Healing Stream Totem](https://wowpedia.fandom.com/wiki/Healing_Stream_Totem) were nerfed to only heal one party/raid member at a time and on a shorter duration with a cooldown.
-Despite this however, things weren't all.
+Despite this however, things weren't all bad.
 Shamans still retained several raid buffs at this point while also gaining a powerful new raid healing cooldown, [Ancestral Guidance](https://wowpedia.fandom.com/wiki/Ancestral_Guidance).
 In its original incarnation, Ancestral Guidance healed all nearby allies for 40% of all direct damage/healing dealt split between them (later changed to 40% of damage and 60% of healing), similar to [Vampiric Embrace](https://wowpedia.fandom.com/wiki/Vampiric_Embrace) and a far cry from what we have today.
 
 Name | 🔥Benefits
 --- | :---:
 [Bloodlust](https://wowpedia.fandom.com/wiki/Bloodlust) / [Heroism](https://wowpedia.fandom.com/wiki/Heroism) | +30% melee, ranged, and spell <br/> casting speed for 40 sec
+[Ancestral Guidance](https://wowpedia.fandom.com/wiki/Ancestral_Guidance) | Damage to Group Healing
 [Burning Wrath](https://wowpedia.fandom.com/wiki/Burning_Wrath) | +10% Spell Power
 [Grace of Air](https://wowpedia.fandom.com/wiki/Grace_of_Air) | +3000 Mastery
 [Elemental Oath](https://wowpedia.fandom.com/wiki/Elemental_Oath) | +5% Spell Crit Chance <br/> (Elemental Only)
 [Unleashed Rage](https://wowpedia.fandom.com/wiki/Unleashed_Rage) | +10% Melee Attack Power <br/> (Enhancement Only)
 [Ancestral Vigor](https://wowpedia.fandom.com/wiki/Restorative_Waves) | Up to +10% Bonus HP <br/> on healed targets <br/> (Restoration Only)
+[Tremor Totem](https://www.wowhead.com/spell=8143/tremor-totem) | Removed 1 Fear, Charm, and Sleep <br/> effect from the raid / 5 sec
+[Healing Stream Totem](https://wowpedia.fandom.com/wiki/Healing_Stream_Totem) | Heals 1 party member <br/> at a time
```

Comment:
One of my guildies roasted me for not including these here even though they weren't all that useful at this point